src/app/project-admin/component/project-admin.component.html

```diff
@@ -0,0 +1,187 @@
+<div *ngIf="project$ | async as project" class="p-4 bg-gray-100 h-full overflow-y-auto flex-1 flex flex-col">
```

Comment:
no need to check for project (see other comments) if you only want to display the page after everything is loaded you can go with 
*ngIf="personalAccessTokens"

Reply:
Removed